apps/site/pages/docs/api-extensions.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: '  Overview'
+---
+
+import { Steps, Step } from 'nextra-theme-docs'
+
+<header className="hero">
+
+# API Extensions
+
+In this guide you will learn how to extends the FastStore API schema.
+</header>
+
+Although the [FastStore API](https://v1.faststore.dev/reference/api/faststore-api) provides a complete [GraphQL schema for ecommerce](https://v1.faststore.dev/reference/api/queries), some stores may need to access other, more specific, information.
+
+You should not add requests to other APIs to your storefront since this may [impact site performance](https://v1.faststore.dev/how-to-guides/faststore-api/fetching-api-data#best-practices-for-fetching-data). For those cases, it is possible to extend the FastStore API schema, adding new data to the existing [queries](https://v1.faststore.dev/reference/api/queries).
+
+---
+
+## Before starting
+
+Note that even though you can add information to the FastStore API schema, you must be careful not to over-fetch data on your pages. See the [best practices for fetching data on your storefront](https://v1.faststore.dev/how-to-guides/faststore-api/fetching-api-data#best-practices-for-fetching-data).
+
+As stores tend to grow, so does the possibility of using data from VTEX APIs that the FastStore API does not expose.
+Another possibility is to use data that is provided by third-party APIs. In both cases, FastStore provides these features by using API Extensions.
+
+Inside the repository, there is a clear separation about what comes from VTEX APIs and what comes from third-party APIs. First, let's take a look in the VTEX APIs Schema extension possibility.
+
+---
+
+## VTEX APIs Schema Extensions
+
+FastStore streamline the way of using data from VTEX APIs that the FastStore API does not expose.
+
+As FastStore uses GraphQL, it's important to define its typeDefs and resolvers when desires creates APIs Extensions by following the steps below
```

Comment:
```suggestion
As FastStore uses GraphQL, it's required to write type definitions and resolvers to fetch the data you need.  You will learn how to do this using API Extensions by following the steps below.
```

Maybe it's worthwhile to add a link to type definition and resolvers for GraphQL.

Reply:
Nice! Thank you. I'll add the 
https://graphql.org/learn/schema/#object-types-and-fields
and https://graphql.org/learn/execution/#root-fields-resolvers
links.